packages/devtools_app/lib/src/survey.dart

```diff
@@ -0,0 +1,166 @@
+import 'dart:convert';
+
+import 'dart:io' if (dart.library.html) 'dart:html';
```

Comment:
this makes me really nervous as a conditional import. dart:html and dart:io are super different. Can we remove this asnd use package:http and similar?

Reply:
The only reason this is here is for use of the `HttpStatus.ok` const, but we could just as easily write `200`.

Reply:
lets write 200 or maybe package:http does or should define that const.
dart static checking falls over badly for conditional imports like this.